dao/src/main/scala/za/co/absa/enceladus/dao/CustomException.scala

```diff
@@ -15,17 +15,16 @@
 
 package za.co.absa.enceladus.dao
 
-abstract class RetryableException(message: String, cause: Throwable) extends Exception(message, cause)
-
+abstract class CustomException(message: String, cause: Throwable) extends Exception(message, cause)
```

Comment:
Does it make sense to rename this? Especially since fields where it's used are called retryableExceptions?

Reply:
The logic & reasoning for this is that before, if an exception was inherited from `RetryableException`, then such error was recoverable. Now this mechanism is completely different and it's not based on checking the exception type. So not just the name was changed, but also the logic (and thus, the name should reflect that).